CONTRIBUTING.md

```diff
@@ -269,18 +299,13 @@ time, run `cargo test` to start generating the snapshots.  The first time you ru
 it will fail, because there's no expected result to compare to.  Let's make the test pass:
 
 We use `insta` for snapshot testing witness results, so after adding/changing a witness, we need
-to update the test outputs. Note that it may contain output for other test crates - this
-is not necessarily an error: see the troubleshooting section for more info.
+to update the test outputs.
 
-There are two ways to update the output:
+> [!TIP]
+> It may contain output for other test crates - this is not necessarily an error:
```

Comment:
this is just me nitpicking on typography, don't mind me 😅
```suggestion
> It may contain output for other test crates — this is not necessarily an error:
```